proto/sedachain/wasm_storage/v1/events.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package sedachain.wasm_storage;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+import "sedachain/wasm_storage/v1/wasm_storage.proto";
+
+option go_package = "github.com/sedaprotocol/seda-chain/x/wasm-storage/types";
+
+
+message EventStoreWasm {
+  string hash = 1;
+  WasmType wasm_type = 2;
+  RequestType request_type = 3;
+
```

Comment:
Try linting using `make proto-lint`. Feel free to change the linter if you dislike it btw..